File: questModule/src/main/java/ru/nekit/android/qls/QuestLockScreenApplication.java
package ru.nekit.android.qls;

import android.app.Application;

import ru.nekit.android.qls.lockScreen.LockScreen;

public class QuestLockScreenApplication extends Application {

    @Override
    public void onCreate() {
        super.onCreate();
        PreferencesUtil.init(this);
        LockScreen.activateForSetupWizard(this);

    }

    @Override
    public void onTerminate() {
        super.onTerminate();
    }
}

File: questApplication/src/main/java/ru/nekit/android/qls/QuestLockScreenApplication.kt
package ru.nekit.android.qls

import android.app.Application
import android.content.Context
import android.support.multidex.MultiDex
import ru.nekit.android.qls.lockScreen.LockScreen

class ktQuestLockScreenApplication : Application() {

    override fun onCreate() {
        super.onCreate()
        PreferencesUtil.init(this)
        LockScreen.activateForSetupWizard(this)
    }

    override fun attachBaseContext(base: Context) {
        super.attachBaseContext(base)
        MultiDex.install(this)
    }
}